catalogue/webapp/pages/item.js

```diff
@@ -5,13 +5,139 @@ import fetch from 'isomorphic-unfetch';
 import { type IIIFManifest, type IIIFCanvas } from '@weco/common/model/iiif';
 import { iiifImageTemplate } from '@weco/common/utils/convert-image-uri';
 import { itemUrl, workUrl } from '@weco/common/services/catalogue/urls';
-import Layout12 from '@weco/common/views/components/Layout12/Layout12';
 import PageLayout from '@weco/common/views/components/PageLayout/PageLayout';
 import Paginator, {
   type PaginatorRenderFunctionProps,
 } from '@weco/common/views/components/RenderlessPaginator/RenderlessPaginator';
 import Control from '@weco/common/views/components/Buttons/Control/Control';
 import { classNames, spacing, font } from '@weco/common/utils/classnames';
+import styled from 'styled-components';
+import Layout12 from '@weco/common/views/components/Layout12/Layout12';
+
+const ItemContainer = styled.div`
+  display: flex;
+  flex-wrap: wrap;
+  width: 100vw;
+  height: 100vh;
+  background: ${props => props.theme.colors.smoke};
+  flex-direction: row-reverse;
+
+  .main {
+    position: relative;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    height: 80%;
+    width: 100%;
+    padding: 24px 6px 60px;
+
+    @media (min-width: 600px) {
+      height: 100%;
+      width: 75%;
+    }
+  }
+
+  .thumbs {
```

Comment:
Are we still trying to be `BEM-y`?
Or, as we are using the new styled components, is getting specific cool again?

Reply:
Not sure – I don't know that BEM makes sense in conjunction with styled components. My (not-necessarily-very-well-thought-out) take is that BEM solves the problem of specificity, but since styled-component styles are scoped to the component, the issue of specificity goes away (unless you make a component so big that it starts to become a problem again, but at that point I think the problem lies with the component design rather than the CSS implementation).

Reply:
I feel it would still help in the sense of if there are any other styles added, specifically `.thumbs` that isn't scoped, we're going to land up with a clash.

We could just remember not to do that, or use something like BEM, which was created as people forget, especially as code bases grow.

Reply:
I don't see where the possible clash would occur? If a different, unscoped, `.thumbs` is added elsewhere, it wouldn't affect these `.thumbs` and vice-versa.

Reply:
…because robo-classes mean these `.thumbs` styles only apply to `.jznmLp.thumbs` elements.

Reply:
Yea, so some stylesheet somewhere has:
```scss
.main {
  background: hotpink;
}
```

and my DOM is
```HTML
<div class="component"><div class="main|>Content</div></div>
```

it will get a pink background no?

Reply:
oh yeah, my mistake – I see where you're coming from.